src/services/export.js

```diff
@@ -815,12 +821,12 @@ const saveEpub = async (page, mode, lang, username, sectionNumber = null) => {
     }
   }
 
-  const backCoverContent = 'generated by Graasp.eu';
-  // title is necessary to not write over cover chapter
-  const backCover = { title: 'Back Cover', data: backCoverContent };
+  // title is needed for epub xml page path
+  // @TODO add data to section's back cover
+  const backCover = { title: 'Back Cover', data: '' };
```

Comment:
@pyphilia, we do not want to add the `title` to the 'Back Cover'. It should only show the text you mentioned above.